api/v1alpha1/linodemachine_types.go

```diff
@@ -64,6 +64,12 @@ type LinodeMachineSpec struct {
 	Tags []string `json:"tags,omitempty"`
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
 	FirewallID int `json:"firewallID,omitempty"`
+	// OSDisk is configuration for the root disk the includes the OS,
```

Comment:
Should these two new specs have the `+optional` marker?

Reply:
I believe kubebuilder defaults to optional by default, at least based on the generated CRD spec on these changes it looks like only sizeMB is required

Reply:
Ah yes I checked the docs and seems that the `omitempty` is equivalent to `+optional`